tf-training/OWNERS

```diff
@@ -1,4 +1,5 @@
 approvers:
   - andreyvelich
   - gaocegege
```

Comment:
@andreyvelich @gaocegege @johnugeorge Can one of you LGTM this since you are already in the OWNERs file?